runsc/cmd/gofer.go

```diff
@@ -114,6 +114,9 @@ func (g *Gofer) Execute(_ context.Context, f *flag.FlagSet, args ...interface{})
 			Fatalf("Error setting up root FS: %v", err)
 		}
 	}
+	if conf.CoverageFilesDirectory != "" {
+		g.applyCaps = false
```

Comment:
Please document why this is needed, and document on the flag definition that it will affect applyCaps.